XbSymbolDatabase.c

```diff
@@ -1283,9 +1289,7 @@ void HLEError(SymbolDatabaseVerifyContext *context, uint16_t buildVersion, char
     (void)strcat(buffer, " : ");
     (void)strncat(buffer, bufferTemp, ret_str_count);
 
-    if (output_func != NULL) {
-        output_func(XB_OUTPUT_MESSAGE_ERROR, buffer);
-    }
+    XbSymbolOutputMessage(XB_OUTPUT_MESSAGE_ERROR, buffer);
```

Comment:
Why was this changed?

Reply:
Nevermind, `XbSymbolOutputMessage` calls `output_func`. I thought it was the other way around.

It's a bit odd, considering `XbSymbolOutputMessage` looks like an external API, given it's name.